.github/workflows/development-workflow.yml

```diff
@@ -6,7 +6,6 @@ on:
         types: [opened, edited, synchronize]
 
 env:
-    PROJECT_ID: ${{ secrets.GKE_PROJECT_ID }}
```

Comment:
Is there a reason this was deleted?

Reply:
Since the deployment is in a different workflow now it's outdated and unnecessary.